transformer/task.py

```diff
@@ -24,6 +24,7 @@ class LocustRequest(NamedTuple):
 
     method: HttpMethod
     url: str
+    name: str
```

Comment:
I would rather see this field at the bottom of the list for the same reasons as exposed for Request.